### ISSUE TITLE
Support Moodle 4.4 and 4.5

### ISSUE THREAD

**Issue by renne** [moodlehq/moodle-tool_migratehvp2h5p#55]
According to https://moodle.org/plugins/tool_migratehvp2h5p/versions Moodle versions are supported up to 4.3.
Please add support for Moodle 4.4 and Moodle 4.5.

**Comment by sarjona** [comment]
Hi @renne!
Thank you for your comment! Support for Moodle 4.4 and 4.5 is on our roadmap, though we are not currently working on it. We’ll be sure to keep you posted on any updates as we move forward. Thank you for your patience!

**Comment by renne** [comment]
@sarjona Is there another solution to convert HVP-courses created by Moodle plugins or external applications to Moodle 4.4/4.5 activitites?

**Comment by sarjona** [comment]
@renne I don't think so. Have you tried if the plugin is working in 4.4/4.5? I haven't checked it but the current version might work for these Moodle versions too ;-)
